tests.js

```diff
@@ -498,10 +515,11 @@ window.Specs = {
 				},
 				"tests": ":root"
 			},
-			":nth-child": {
+			":nth-child()": {
 				"links": {
 					"tr": "#nth-child-pseudo",
-					"dev": "#nth-child-pseudo"
+					"dev": "#nth-child-pseudo",
+					"mdn": ":nth-child"
```

Comment:
This looks like it can be generated (see comment about functions). Same about many of the following functional pseudo-classes.